src/main/java/step4/RacingEntry.java

```diff
@@ -0,0 +1,37 @@
+package step4;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RacingEntry {
+
+    private final List<Car> cars;
+
+    public RacingEntry(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public List<Car> getCars() {
+        return this.cars.subList(0, this.cars.size());
+    }
+
+    public List<Car> findWinners() {
+        final int maxLocation = findMaxLocation();
+
+        return this.cars.stream()
+                .filter(car -> isSameLocation(maxLocation, car.getLocation()))
+                .collect(Collectors.toList());
```

Comment:
현재 for문안에 if문이 포함되어 있어서  2depth가 되었습니다.

프로그래밍 요구사항을 확인해보시면 아래와 같이 명시되어 있습니다. 🙇🏻‍♀️
**indent(인덴트, 들여쓰기) depth를 2를 넘지 않도록 구현한다. 1까지만 허용한다.
예를 들어 while문 안에 if문이 있으면 들여쓰기는 2이다.
힌트: indent(인덴트, 들여쓰기) depth를 줄이는 좋은 방법은 함수(또는 메소드)를 분리하면 된다.** 
```suggestion
      for (Car car : this.cars) {
            if (isSameLocation(maxLocation, car.getLocation())) {
                list.add(car);
            }
        }
```

Reply:
filter 메소드의 람다 비교구문이 isSameLocation메소드로 분리되어 있어서 2depth가 되지는 않을것 같다고 생각합니다ㅎㅎ